.github/workflows/docs.yml

```diff
@@ -11,10 +11,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - name: Setup node 😻
+        uses: actions/setup-node@v3
         with:
-          node-version: 16.x
-          cache: "yarn"
+          node-version-file: '.node-version'
+          cache: 'yarn'
```

Comment:
Err, I think the quotes changed while copy-and-pasting. Can change it back if we prefer.